website_livechat_firstname/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
```

Comment:
In version 9, this file is actually called `__openerp__.py`. Changing this should fix your build error